docs/fundamentals/executor/hub/push-executor.md

```diff
@@ -28,6 +28,11 @@ You can then visit [the Hub portal](https://hub.jina.ai), click on the "Recent"
 If no `--public` or `--private` argument is provided, then it is **public** by default.
 ````
 
+````{admonition} Note
+:class: note
+If you are logged in to the Hub using our CLI tools (`jina auth login` or `jcloud login`), you can push and pull your executors without `SECRET`.
+````
+
```

Comment:
This is very clear to me, I like the Note box! But could you move it up above "You can visit...", and below "Please keep them carefully."?
That way the different boxes are not all squashed together, and this note is actually closer to where the section talks about secret management.
If you try that and it looks bad just keep it the way it is now I would say.

Reply:
great suggestions. I like it better!

![Screenshot 2022-06-28 at 10 22 54](https://user-images.githubusercontent.com/492616/176131338-602e0f42-255d-4011-819d-22515547f000.png)